src/tabbar.h

```diff
@@ -65,7 +65,6 @@ public slots:
     QStackedWidget*     mp_stackedWidget;
     int                 m_settingsIndex;
     QScopedPointer<FullScreenWindow> m_fullScreenWindow;
-
```

Comment:
This empty line visually separates properties definition from methods. So let's keep it.